### PR TITLE
Add new attributes to send-email action

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

**Comment by vova07** [comment]
please review

File: spec/definitions/Rules/RulesEmailNotification.yaml [hunk 1]
```diff
@@ -3,6 +3,12 @@ properties:
   credentialHash:
     type: string
     description: SMTP or Mailgun credential identifier string.
+  name:
```
Comment (slavcodev) [inline spec/definitions/Rules/RulesEmailNotification.yaml]: Please consider to if they both optional, to add defaults, or maybe name is mandatory?

Comment (vova07) [reply spec/definitions/Rules/RulesEmailNotification.yaml]: Both are optional. And there are no needs for default values.

**Comment by vova07** [comment]
please re-review

File: spec/definitions/Rules/RulesEmailNotification.yaml [hunk 2]
```diff
@@ -3,6 +3,12 @@ properties:
   credentialHash:
     type: string
     description: SMTP or Mailgun credential identifier string.
+  title:
+    type: string
+    description: The title of the message. (This title will appear in reports.)
+  version:
+    type: string
+    description: The message version. (By default is empty, but if you make multiple versions, you can name or number them to distinguish them.)
```
Comment (2e3s) [inline spec/definitions/Rules/RulesEmailNotification.yaml]: It's not auto-incremented? Why have a title and version apart?

**Pull request merged**